docs/spec/v2beta2/helmreleases.md

```diff
@@ -0,0 +1,1587 @@
+a [Helm uninstall action](https://helm.sh/docs/helm/helm_uninstall/). This
+configuration applies to the [install remediation](#install-remediation), and
+when the [upgrade remediation strategy](#upgrade-remediation) is set to
+`uninstall`.
+
+The field offers the following subfields:
+
+- `.timeout` (Optional): The time to wait for any individual Kubernetes
+  operation (like Jobs for hooks) during the uninstalltion of the release.
+  Defaults to the [global timeout value](#timeout).
+- `.deletionPropagation` (Optional):
+ `.disableHooks` (Optional): Prevents [chart hooks](https://helm.sh/docs/topics/charts_hooks/)
+  from running during the uninstallation of the release. Defaults to `false`.
+- `.disableWait` (Optional): Disables waiting for resources to be deleted after
+  uninstalling the release. Defaults to `false`.
+- `.keepHistory` (Optional): Instructs Helm to remove all associated resources
+  and mark the release as deleted, but to retain the release history. Defaults
+  to `false`.
+
+### Drift detection
+
+`.spec.driftDetection` is an optional field to enable the detection (and
+correction) of cluster-state drift compared to the manifest from the Helm
+storage.
+
+When `.spec.driftDetection.mode` is set to `warn` or `enabled`, and the
+desired state of the HelmRelease is in-sync with the Helm release object in
+the storage. The controller will compare the manifest from the Helm storage
+with the current state of the cluster using a
+[server-side dry-run apply](https://kubernetes.io/docs/reference/using-api/server-side-apply/).
+
+If this comparison detects a drift (either due to a resource being created
+or modified during the dry-run), the controller will emit a Kubernetes Event
+with a short summary of the detected changes. In addition, a more extensive
+[JSON Patch](https://datatracker.ietf.org/doc/html/rfc6902) summary is logged
+to the controller logs (with `--log-level=debug`).
+
+#### Drift correction
+
+When `.spec.driftDetection.mode` is set to `enabled`, the controller will
+attempt to correct the drift by creating and patching the resources based on
+the server-side dry-run result.
+
+At the end of the correction attempt, it will emit a Kubernetes Event with a
+summary of the changes it made and any failures it encountered. In case of a
+failure, it will continue to detect and correct drift until the desired state
+has been reached, or a new Helm action is triggered (due to e.g. a change to
+the spec).
+
+#### Ignore rules
+
+`.spec.driftDetection.ignore` is an optional field to provide
+[JSON Pointers](https://datatracker.ietf.org/doc/html/rfc6901) to ignore while
+detecting and correcting drift. This can for example be easeful when Horizontal
+Pod Autoscaling is enabled for a Deployment, or when a Helm chart has hooks
+which mutate a resource.
+
+```yaml
+spec:
+  driftDetection:
+    mode: enabled
+    ignore:
+      - paths: ["/spec/replicas"]
+```
+
+**Note:** It is possible to achieve a likewise behavior as using
```

Comment:
```suggestion
**Note:** It is possible to replicate the behavior of using
```
maybe?

Reply:
I feel that "replication" signals a 1:1 copy of behavior, which isn't really the case. It's "kind of" the same.